libraries/botbuilder-lg/src/staticChecker.ts

```diff
@@ -27,13 +27,20 @@ export class StaticChecker extends AbstractParseTreeVisitor<Diagnostic[]> implem
     private currentTemplate: Template;
     private _expressionParser: ExpressionParserInterface;
 
+    /**
+     * Creates a new instance of the StaticChecker class.
+     * @param templates Templates to be checked.
+     */
     public constructor(templates: Templates) {
         super();
         this.templates = templates;
         this.baseExpressionParser = templates.expressionParser || new ExpressionParser();
     }
 
-    // Create a property because we want this to be lazy loaded
+    /**
+     * @private
+     * Creates a property because we want this to be lazy loaded.
+     */
```

Comment:
If this @private do you want documentation?

Reply:
Hi @chrimc62, we put it because it exists on DotNet when we search for it, and we always aim for parity.